platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/network/TopologyTest.java

```diff
@@ -107,7 +107,7 @@ private static void testRandomGraphWithSets(
     @MethodSource({"failing", "topologicalVariations", "fullyConnected"})
     void testRandomGraphs(final int numNodes, final int numNeighbors, final long seed) throws Exception {
         System.out.println("numNodes = " + numNodes + ", numNeighbors = " + numNeighbors + ", seed = " + seed);
-        final RandomGraph randomGraph = new RandomGraph(numNodes, numNeighbors, seed);
+        final RandomGraph randomGraph = new RandomGraph(new Random(), numNodes, numNeighbors, seed);
```

Comment:
print random seed?

Reply:
Agreed, that is a better pattern. Change made.